pkgs/development/python-modules/mkdocs-jupyter/default.nix

```
@@ -0,0 +1,55 @@
+{ buildPythonPackage
+, fetchpatch
+, fetchPypi
+, ipykernel
+, jupytext
+, lib
+, lxml
+, mkdocs
+, mkdocs-material
+, nbconvert
+, pygments
+, pytestCheckHook
+, pytest-cov
+, pyyaml
+, requests
+, stdlib-list
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-jupyter";
+  version = "0.22.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-WFzGm+pMufr2iYExl43JqbIlCR7UtghPWrZWUqXhIYU=";
+  };
+
+  postPatch = ''
+  substituteInPlace mkdocs_jupyter/tests/test_base_usage.py \
```

There is a missing indentation (maybe running `nixpkgs-fmt` on this file might fix this issue).
```suggestion
    substituteInPlace mkdocs_jupyter/tests/test_base_usage.py \
```

---

done